Source/Extensions/Blazorise.Splitter/Splitter.razor.cs

```diff
@@ -0,0 +1,255 @@
+        await createInstanceLock.WaitAsync();
+
+        try
+        {
+            if ( JSSplitInstance is not null )
+                await JSSplitInstance.InvokeVoidAsync( "destroy" );
+
+            if ( splitterSections.Count > 0 )
+            {
+                JSSplitInstance = await JSModule.InitializeSplitter( splitterSections.OrderBy( x => x.Index ).Select( x => x.ElementRef ), new SplitterOptions
+                {
+                    Sizes = splitterSections.Any( x => x.Size != null )
+                        ? splitterSections.OrderBy( x => x.Index ).Select( x => new JavascriptNumber( x.Size ?? 0 ) ).ToArray()
+                        : null,
+                    MinSize = splitterSections.OrderBy( x => x.Index ).Select( x => new JavascriptNumber( x.MinSize ) ).ToArray(),
+                    MaxSize = splitterSections.OrderBy( x => x.Index ).Select( x => new JavascriptNumber( x.MaxSize ) ).ToArray(),
+                    ExpandToMin = ExpandToMin,
+                    GutterSize = GutterSize,
+                    GutterAlign = GutterAlign,
+                    SnapOffset = splitterSections.Any( x => x.SnapOffset > 0 )
+                        ? splitterSections.OrderBy( x => x.Index ).Select( x => new JavascriptNumber( x.SnapOffset ) ).ToArray()
+                        : 0,
+                    DragInterval = DragInterval,
+                    Direction = Direction,
+                    Cursor = Cursor,
+                }, new SplitterGutterOptions
+                {
+                    BackgroundImage = GutterBackgroundImage,
+                } );
+
+                recreateInstance = false;
+            }
+        }
+        finally
+        {
+            createInstanceLock.Release();
+        }
+    }
+
+    #endregion
+
+    #region Properties
+
+    /// <summary>
+    /// Gets or sets the <see cref="JSSplitModule"/> instance.
+    /// </summary>
+    protected JSSplitModule JSModule { get; private set; }
+
+    /// <summary>
+    /// Gets or sets the reference to the JS split instance
+    /// </summary>
+    protected IJSObjectReference JSSplitInstance { get; private set; }
+
+    /// <summary>
+    /// Gets or sets the JS runtime.
+    /// </summary>
+    [Inject] private IJSRuntime JSRuntime { get; set; }
+
+    /// <summary>
+    /// Gets or sets the version provider.
+    /// </summary>
+    [Inject] private IVersionProvider VersionProvider { get; set; }
+
+    /// <summary>
+    /// When the split is created, if ExpandToMin is true, the minSize for each element overrides the percentage value from the sizes option.
+    /// </summary>
+    [Parameter] public bool? ExpandToMin { get; init; }
+
+    /// <summary>
+    /// Gutter size in pixels. Defaults to 10.
+    /// </summary>
+    [Parameter] public double GutterSize { get; init; } = 10;
+
+    /// <summary>
+    /// Defines the custom background image for the gutter element.
+    /// </summary>
+    [Parameter] public string GutterBackgroundImage { get; init; }
```

Comment:
Can we have an example of this on the docs? I think it might be a common operation.

Reply:
Additionally tell the user what's expected here, a base64? a path?